tools/test/topology/test-playback.m4

```diff
@@ -152,8 +152,8 @@ SectionGraph."pipe-sof-fourth-pipe" {
 
 # DAI configuration
 
-# Use 3 periods for SRC DAI buffer, otherwise 2 periods
-ifelse(TEST_PIPE_NAME, `src', `define(TEST_DAI_PERIODS, `3')', `define(TEST_DAI_PERIODS, `2')')
+# Use 4 periods for SRC DAI buffer, otherwise 2 periods
```

Comment:
These are TBD, maybe 3 & 3 could work. I'll test. I did this before noticing the stage1 and stage2 conditions.

Reply:
Yes, the existing 3 source and 3 sink buffers were sufficient. I removed this topology commit.